wgpu-hal/src/dx12/adapter.rs

```diff
@@ -711,6 +733,11 @@ impl super::Adapter {
                     } else {
                         0
                     },
+
+                    // See https://microsoft.github.io/DirectX-Specs/d3d/ViewInstancing.html#maximum-viewinstancecount
+                    // This is really frickin annoying, 6 (probably 8) for cube mapping would be really nice. But they
+                    // arbitrarily chose 4, eliminating tons of use cases.
```

Comment:
```suggestion
```

language plz! also, those hardworking engineers certainly probably good reason, so let's rant & not complain _at_ them ;-).

Also, the comment seems to be in the wrong spot anyways - the limit for 4 is further up